docs/tasks/configure-pod-container/security-context.md

```diff
@@ -345,6 +347,8 @@ label given to all Containers in the Pod as well as the Volumes.
 * [Security Contexts design document](https://git.k8s.io/community/contributors/design-proposals/security_context.md)
 * [Ownership Management design document](https://git.k8s.io/community/contributors/design-proposals/volume-ownership-management.md)
 * [Pod Security Policies](/docs/concepts/policy/pod-security-policy/)
+* [AllowPrivilegeEscalation design
+  document](https://git.k8s.io/community/contributors/design-proposals/no-new-privs.md)
```

Comment:
The link is broken now, here is a working one: https://github.com/kubernetes/community/blob/master/contributors/design-proposals/auth/no-new-privs.md